topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md

```diff
@@ -0,0 +1,767 @@
+---
+layout: tutorial_hands_on
+
+title: Generating a cell matrix using Alevin
+subtopic: single-cell
+priority: 9
+
+zenodo_link: ''
+questions:
+- I have some single cell fastq files I want to analyse. Where do I start?
+objectives:
+- Repeat matrix generation for any droplet-based single cell sequencing data
+- Apply data combination and metadata editing for particular experimental designs
+- Interpret quality control (QC) plots to make informed decisions on cell thresholds
+- Find relevant information in GTF files for the particulars of their study, and include this in data matrix metadata
+time_estimation: 3H
+key_points:
+- Create a scanpy-accessible AnnData object from fastq files, including relevant cell and gene metadata
+- Combine multiple samples and label according to study design
+requirements:
+-
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-intro
+        - scrna-umis
+tags:
+- single-cell
+- 10x
+contributors:
+- nomadscientist
+- pinin4fjords
+
+---
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+This tutorial will take you from raw fastq files to a cell x gene data matrix in AnnData format. What's a data matrix, and what's AnnData format? Well you'll find out! Importantly, this is the first step in processing single cell data in order to start analysing it. Currently you have a bunch of strings of ATGGGCTT etc. in your sequencing files, and what you need to know is how many cells you have and what genes appear in those cells. In the second part of this tutorial, we will also look at combining fastq files and adding in metadata (for instance, SEX or GENOTYPE) for analysis later on. These steps are the most computationally heavy in the single cell world, as you're starting with 100s of millions of reads, each 4 lines of text. Later on in analysis this data becomes simple gene counts such as 'Cell A has 4 GAPDHs', which is a lot easier to store! Because of this data overload, we have downsampled the fastq files to speed up the analysis a bit. Saying that, you're still having to map loads of reads to the massive murine genome, so get yourself a cup of coffee and prepare to analyse!
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# 1: Generating a matrix
```

Comment:
Would you mind terribly removing the numbers from the section headings? The ToC is always on the left, so, maybe a bit harder to get lost since you've give the sections good names already.

Reply:
Do you reckon the sub-sections as well?

Reply:
yes, please, if you don't mind. I know some folks like their numbered sections, we can definitely look into doing that automatically in the table of contents maybe? If that'd be interesting?

We just want to save the work of updating every single section number if someone needs to add/remove/rearrange things at any point in the future.

Reply:
Absolute fair point there. I did notice it seemed to auto-update the numbers, because I screwed up the labels but it corrected them in render, which I had thought was a fancy feature! I personally like the numbers because it helps me as a user know where I am, but the EBI's train online team would agree with you rather than me on that!  I'm removing them now...